bcgov/HOUS-permit-portal
# Add review manager flow for retrieving and reviewing template changes

## Description

Add template diff events that generate on template publish.
These create notifications that prompt the review manager to open the template edit page with ?compare=true
This triggers the load of the template diff which uses the previous template version by default.

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-1234

## Steps to QA

Log in as super admin
Edit a template (add, remove, change requirements) and (force) publish the change
log in as a review manager
open notifications, click go link
See the yellow sidebar load, click through the changes to be brought to the corresponding blocks if still available
